### DISCUSSION TITLE
Welcome to feroxbuster Discussions!

### DISCUSSION THREAD

**Discussion by epi052** [epi052/feroxbuster#149] [General]
## 👋 Welcome!
  Giving this Discussions thing a try :shrug:. Please use this space to:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Some way cooler thing I haven't considered.

**Comment by 0xVenus** [comment]
I'm getting"bus error" after installation

**Reply by epi052** [reply]
please see reply in the new thread 

https://github.com/epi052/feroxbuster/discussions/555#discussion-4043647

**Comment by rtificial-flava** [comment]
I've possibly found a bug. If you use a wordlist of considerable size(in my case 1+gig wordlist) it will make feroxbuster crash/stop. Is there a way around this or can this be patched later on?  All my other wordlists work great.
